tests/std/tests/P0898R3_concepts/test.cpp

```diff
@@ -2352,7 +2352,7 @@ namespace test_object_concepts {
     STATIC_ASSERT(!test_semiregular<ExplicitMoveAbomination>());
     STATIC_ASSERT(!test_semiregular<ExplicitCopyAbomination>());
     STATIC_ASSERT(!test_semiregular<CopyOnlyAbomination>());
-    STATIC_ASSERT(test_semiregular<SemiregularType>()); // Who woulda guessed?
+    STATIC_ASSERT(test_semiregular<SemiregularType>()); // Who would have guessed?

```

Comment:
This change in tone kills the joke (well, what there is of it anyway). I'd rather revert this change or strike the comment completely.

Reply:
I'd be in favor of striking the comment.

Reply:
```suggestion
    STATIC_ASSERT(test_semiregular<SemiregularType>());
```